src/components/FormSelectionContainer/FormSelection/FormSelection.tsx

```diff
@@ -29,11 +29,11 @@ export const FormSelection: FunctionComponent<FormSelection> = ({ className, con
         <Wrapper>
           <ProgressBar step={1} />
           <Title className="mb-8">Choose Document Type to Issue</Title>
-          <div className="flex w-full">
+          <div className="flex w-full flex-wrap justify-start">
             {config.forms.map((form: FormTemplate, index: number) => {
```

Comment:
think no need `w-full`, did nothing in this case

Reply:
ok will remove it.